sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -656,6 +806,16 @@ public void processElement(ProcessContext context) throws Exception {
                   || (currentBatch.getMoreResults() == NOT_FINISHED));
         }
       }
+
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
```

Comment:
You should use .include instead - see https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/display/DisplayData.java#L168